destroy_webserver.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if ( multipass info relativepath 2> /dev/null )
+then
+    echo "Deleting the vm relativepath..."
+    multipass delete --purge relativepath
+else
+    echo "the vm relative has already been deleted!"
```

Comment:
nit: "**T**he VM relative**path**..."
nit: replace '...has already been deleted!' with 'does not exist'
Sidenote, these would be comments made by a PM during a demo. Or a dev who minored in English.